sdk/core/core-util/package.json

```diff
@@ -76,7 +76,8 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^2.0.0",
-    "tslib": "^2.6.2"
+    "tslib": "^2.6.2",
+    "@typespec/ts-http-runtime": "^0.2.1"
```

Comment:
Now that I think about it, should we pin to `0.2.1` or `~0.2.1`? We don't usually float non-GA versions but we may be more lax here since it's all owned by us. Thoughts?

Reply:
From a technical point of view caret is still okay here -- in the 0.x range, [caret matches the patch number only](https://devhints.io/semver). It's different to a beta where using a caret will pull in any future betas, breaking or not, which is definitely something to be avoided.

If we bump the minor version number while doing breaking changes before 1.0.0 (which is the plan) floating should still always pull in a compatible version. I do think that since this is all in our control keeping it floating is fairly safe, and means we can ship new non-breaking features/bugfixes in unbranded without re-releasing the rest of Core, but am curious what others think too.